profile/README.md

```diff
@@ -1,3 +1,30 @@
 # California Cropland Monitoring and Modeling Framework
 
 ## About
+
+The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
```

Comment:
"to generate historical inventories and future projections"?

Reply:
@serbinsh its not clear - what is the question? Does the wording seem odd or confusing? 

Goal is to convey that we will generate:
1. Annual inventories of SOC and GHG fluxes and
2. Future projections of SOC and GHG fluxes under different agronomic management and climate scenarios

Reply:
my idea here was to be more specific since just saying inventories doesnt exactly clarify the timeline or if this means from now on? for a historical period?  so historical was meant to be added to clarify

Reply:
well, it is only 'historical' in the sense that each year we are creating an inventory ('reanalysis???') for the year that has just ended.

Reply:
```suggestion
The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate current year inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
```